components/uploadButton.js

```diff
@@ -66,19 +92,23 @@ function UploadButton({ navigation, image }) {
         await uploadToFirebase(image);
 
         // 2. store image locally in gallery
-        db.transaction((tx) => {
-          tx.executeSql(
-            "insert into gallery (uri) values (?)",
-            [image.image_uri],
-            () => { },
-            (_, error) => console.log(error)
-          );
-          console.log(image.image_uri);
-        });
+        // db.transaction((tx) => {
```

Comment:
should we delete this code? or is there a reason its commented / we want to use it later?

Reply:
Hm this is for the gallery originally... Should I keep it in case we want to implement the gallery at some point?